tools/depends/Makefile.include.in

```diff
@@ -98,6 +99,15 @@ NATIVE_LDFLAGS=@host_includes@ -L@prefix@/@tool_dir@/lib
 NATIVE_CPPFLAGS=@host_includes@ -I@prefix@/@tool_dir@/include
 NATIVE_CXXFLAGS=@host_includes@ -I@prefix@/@tool_dir@/include
 
+VERSION.TXT := $(CMAKE_SOURCE_DIR)/version.txt
+APP_NAME=$(shell awk '/APP_NAME/ {print tolower($$2)}' $(VERSION.TXT))
+
+PYTHON_VERSION=3.8
```

Comment:
can this be evaluated instead of hardcoded?

Reply:
I don't know how you would. I guess can grep in each makefile once python is built. This just relocated from the site.config into the makefile.include, as my understanding is autoconf will use the site.config, but things that don't use autoconf (ie our existing python modules) don't use site.config vars

Reply:
Ive tried to update any/all other hardcoded locations of the python version number, so everything else now only relies on this as a point of update. Best i can come up with for now.